docs/howto/openstack/nova/move-server-between-regions.md

```diff
@@ -0,0 +1,221 @@
+
+    ```bash
+    openstack server show -c volumes_attached <server_id>
+    ```
+
+    Output:
+
+    ```plain
+    +------------------+-------------------------------------------+
+    | Field            | Value                                     |
+    +------------------+-------------------------------------------+
+    | volumes_attached | id='14942b23-....-....-....-............' |
+    |                  | id='34e2ca96-....-....-....-............' |
+    |                  | id='68756558-....-....-....-............' |
+    +------------------+-------------------------------------------+
+    ```
+
+    If there are multiple volumes attached, the first volume in the list is the server system volume. Copy this ID.
+
+## Creating a copy of a volume
+
+To make an image out of a server, first you have to make a copy of the system volume, by using the following command:
+
+```bash
+openstack volume create --source <volume_id> <new_volume_name>
+```
+
+You will get a printout showing you information about the created volume, such as `source_volid` which is the ID of your server's system volume and `id` of this new volume, that you will use in the next step to create the image.
+
+```plain
++---------------------+--------------------------------------+
+| Field               | Value                                |
++---------------------+--------------------------------------+
+| attachments         | []                                   |
+| availability_zone   | nova                                 |
+| bootable            | true                                 |
+| consistencygroup_id | None                                 |
+| created_at          | 2022-11-10T15:12:11.821647           |
+| description         | None                                 |
+| encrypted           | False                                |
+| id                  | 9e171aa3-....-....-....-............ |
+| multiattach         | False                                |
+| name                | <new_volume_name>                    |
+| properties          |                                      |
+| replication_status  | None                                 |
+| size                | 20                                   |
+| snapshot_id         | None                                 |
+| source_volid        | 14942b23-....-....-....-............ |
+| status              | creating                             |
+| type                | default                              |
+| updated_at          | None                                 |
+| user_id             | ................................     |
++---------------------+--------------------------------------+
+```
+
+## Creating an image of a volume
+
+Then create the image, by using the following command:
+
+```bash
+openstack image create --volume <volume_id> <new_image_name>
+```
+
+> Substitute `<volume_id>` with the ID from the newly created volume in the previous step.
```

Comment:
Don't reuse placeholder names. Earlier, you used `<volume_id>` to refer to the UUID of the server's volume; now you're using it to refer to the UUID of a copy. Use something like `<source_volume_id>` and `<copy_volume_id>` instead.